### ISSUE TITLE
Function send Notification and send Email of the Automation not working on account  user and staff

### ISSUE THREAD

**Issue by buihuuduc** [blynkkk/beta#154]
I use automation for sensor input with condition " device A sensor input is less than 100" action Send Notification and send Email, turn off the relay. It is working  well in acc admin, but not in acc user and staff. I use 1 template id and 3 device.

**Comment by doom369** [comment]
You are right, for now notifications can receive only the owner/creator of automation. We'll extend this to allow to select all users.

**Comment by buihuuduc** [comment]
Noted. Thanks for your feedback.
